src/test/java/org/vertx/eventbus3/bridge/TestServer.java

```diff
@@ -0,0 +1,73 @@
+package org.vertx.eventbus3.bridge;
+
+import org.vertx.java.core.Handler;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class TestServer extends Thread {
+
+  Handler<String> lineHandler;
+
+  private void process(String line) {
+    if (line.startsWith("INFO: ")) {
+      if (lineHandler != null) {
+        lineHandler.handle(line.substring(6));
+      } else {
+        System.err.println("Unexpected: " + line.substring(6));
+      }
+    }
+  }
+
+  @Override
+  public void run() {
+    try {
+
```

Comment:
Oh I see,

what we can do is to configure maven to download it in target/lib (or whatever) and if people want to run the test they would need to execute mvn initialize first (downloading the jar). This would be documented in the README file.

The main advantage: easy to bump versions.